tests/filtering/exclude.ini

```diff
@@ -4,12 +4,5 @@ bar
 schema_name_20_chars
 
 [exclude-table]
-; public.actor
-; public.category
-; public.film
-; public.film_actor
-; public.film_category
-; public.language
-; public.rental
```

Comment:
May I know why this has been removed? I think this has been kept to check the commenting aspect of ini file.

Reply:
Good catch! I initially removed the commented lines, thinking they were leftovers. However, I understand now that they may serve a purpose in checking the commenting aspect of the INI file. I added back the commented lines and included a clear comment on top. [5a9d1c9](https://github.com/dimitri/pgcopydb/pull/614/commits/5a9d1c90ce0cd3e125bf2df958d6bef0d0c9276f)